read-fonts/src/tables/variations.rs

```diff
@@ -411,28 +411,35 @@ impl<'a> PackedDeltas<'a> {
     }
 }
 
+/// The type, notably influence how many bytes per entry, of value in a delta run
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum DeltaRunValueType {
```

Comment:
`RunType` also feels like a reasonable name? Or `RunValueType`? 🤷

Reply:
Everything else had the Delta prefix so it feels weird without that so ... DeltaRunType perhaps. Will try that.